lib/iex/lib/iex/helpers.ex

```diff
@@ -394,6 +394,11 @@ defmodule IEx.Helpers do
 
   @doc """
   Prints information about the given data type.
+  
+  ## Examples
+
+      i(Int)
+      i(Atom)
```

Comment:
With these examples it seems like `i/1` only works on modules though? Also, `h/1` and `b/1` indeed have examples, but I think that's more justified because they don't work on "valid Elixir terms" (e.g., `h(Enum.reduce/1)`), I'm not sure examples add to the value of the documentation for `i/1` though 😕